src/main/js/pullRequest.ts

```diff
@@ -140,13 +140,23 @@ export function check(pullRequest: PullRequest) {
   return apiClient.post((pullRequest._links.mergeCheck as Link).href, {}).then(response => response.json());
 }
 
+/**
+ * @deprecated
+ */
 export function getDefaultCommitDefaultMessage(url: string) {
   return apiClient
     .get(url)
     .then(response => response.text())
     .catch(err => "");
 }
```

Comment:
Whatfor do we have to keep this one? I don't think that we should consider plugin javascript api as public. The rest api is another case, because someone may have used this somehow, but ... can we even import javascript from a plugin in another plugin?